Cargo.toml

```diff
@@ -878,6 +879,7 @@ doc-scrape-examples = true
 name = "2D Wireframe"
 description = "Showcases wireframes for 2d meshes"
 category = "2D Rendering"
+# Wireframes are not supported by WebGL
```

Comment:
I think it's technically more accurate to that PolygonMode::Line is not supported. If we used something like gizmos rendering to draw wireframes it would work in the web.

Doesn't need to change though because at the end of the day that means our current wireframe impl doesn't work in the web.